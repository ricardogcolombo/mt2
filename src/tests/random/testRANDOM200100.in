200 100
1 144 130 36 38
1 160 122 180 8
1 186 128 20 117
1 44 60 3 74
1 179 106 74 59
1 191 55 139 101
1 38 61 193 3
1 124 32 43 57
1 140 7 130 95
1 139 30 63 15
1 140 67 155 108
1 200 75 39 72
1 79 106 137 130
1 34 86 150 94
1 184 49 38 70
1 152 119 3 113
1 11 3 22 57
1 163 64 160 83
1 179 123 137 39
1 41 40 196 39
1 199 103 109 59
1 132 80 169 44
1 185 59 63 113
1 2 28 42 53
1 163 120 37 84
1 77 103 6 124
1 114 101 64 128
1 125 97 142 23
1 116 90 173 45
1 26 10 85 24
1 71 34 38 44
1 66 33 94 45
1 134 72 157 63
1 187 55 38 34
1 129 10 53 37
1 168 17 65 56
1 194 45 25 105
1 118 39 130 56
1 79 11 34 61
1 45 20 11 12
1 60 48 30 94
1 176 124 125 82
1 119 78 11 1
1 173 126 77 2
1 131 22 58 28
1 30 6 164 63
1 163 72 116 123
1 54 110 164 111
1 176 1 39 113
1 51 53 144 79
1 177 54 182 94
1 98 26 16 65
1 71 117 90 51
1 157 78 54 92
1 130 112 84 19
1 177 43 117 13
1 43 80 7 37
1 7 22 85 45
1 103 90 149 84
1 196 111 101 130
1 156 37 153 27
1 110 25 96 28
1 198 80 192 49
1 9 12 103 19
1 88 53 24 105
1 75 74 35 10
1 139 39 83 120
1 101 54 147 97
1 163 124 96 25
1 23 5 146 68
1 156 100 154 0
1 167 24 80 64
1 18 91 26 78
1 159 72 171 103
1 49 31 6 66
1 129 74 109 111
1 183 80 12 60
1 80 46 41 130
1 121 54 136 75
1 142 44 42 66
1 191 85 41 115
1 130 108 76 18
1 41 65 200 83
1 170 47 143 93
1 151 31 146 41
1 93 8 52 51
1 36 20 8 45
1 193 113 33 97
1 3 22 122 55
1 63 18 127 84
1 44 84 136 39
1 65 51 150 123
1 146 18 8 75
1 100 43 124 19
1 177 101 127 68
1 159 40 178 74
1 51 44 148 23
1 49 7 56 57
1 19 0 199 67
1 97 92 163 87
