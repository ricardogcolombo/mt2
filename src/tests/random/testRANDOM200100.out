-nan
0.000000
0.333333
-nan
-nan
1.000000
0.000000
1.000000
0.000000
-nan
0.000000
0.000000
-nan
-nan
-nan
1.000000
-nan
1.000000
0.000000
0.000000
-nan
1.000000
0.000000
1.000000
1.000000
0.000000
-nan
-nan
-nan
0.500000
-nan
-nan
0.000000
0.500000
0.000000
0.000000
0.000000
0.750000
0.500000
-nan
0.750000
1.000000
1.000000
0.500000
1.000000
-nan
-nan
-nan
0.000000
-nan
0.500000
1.000000
1.000000
0.500000
-nan
1.000000
-nan
1.000000
-nan
0.000000
-nan
-nan
0.333333
1.000000
0.500000
0.000000
-nan
-nan
-nan
-nan
0.500000
-nan
-nan
0.000000
1.000000
0.000000
0.000000
-nan
0.000000
0.500000
-nan
-nan
1.000000
0.000000
1.000000
-nan
-nan
0.000000
-nan
0.000000
-nan
-nan
0.000000
1.000000
-nan
0.500000
1.000000
0.000000
-nan
1.000000
0.500000
-nan
1.000000
-nan
-nan
-nan
-nan
-nan
0.500000
0.000000
-nan
-nan
-nan
0.000000
-nan
1.000000
0.000000
0.000000
1.000000
-nan
0.000000
1.000000
-nan
0.000000
0.500000
-nan
0.500000
-nan
0.000000
1.000000
0.000000
1.000000
-nan
1.000000
-nan
0.500000
0.500000
-nan
0.666667
0.000000
-nan
0.000000
1.000000
1.000000
-nan
0.666667
1.000000
0.000000
0.000000
1.000000
0.000000
1.000000
0.000000
0.000000
1.000000
1.000000
0.000000
-nan
0.000000
1.000000
-nan
-nan
0.400000
1.000000
-nan
-nan
0.000000
0.000000
0.000000
0.000000
1.000000
-nan
0.500000
-nan
-nan
0.500000
0.666667
1.000000
1.000000
0.000000
-nan
1.000000
1.000000
0.000000
0.000000
1.000000
1.000000
-nan
-nan
-nan
0.000000
0.000000
0.500000
0.000000
-nan
0.000000
-nan
1.000000
1.000000
1.000000
